include/signalrclient/hub_connection_builder.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#pragma once
+
+#include "_exports.h"
+#include "hub_connection.h"
+#include <memory>
+#include "websocket_client.h"
+#include "http_client.h"
+
+namespace signalr
+{
+    class hub_connection_builder
```

Comment:
This is the main thing to look at for now. Let's decide how we want the builder to look.